README.md

```diff
@@ -129,6 +129,16 @@ Using the ruby bundler:
 bundle exec ruby ./bin/mindee.rb invoice path/to/your/file.ext
 ```
 
+### 🧪 Experimental Features
+
+#### PDF fix
+
+Some pdf files might appear fine on your computer, but not on the server.
+This _experimental_ feature attempts to fix the file's header information before sending it to the server.
+
+> ⚠️ **Warning**: This feature copies your file and then **alters** it. The original file will be left alone, but the copy might get partially corrupted, and improperly parsed as a result. Use at your own discretion.
+
+
```

Comment:
This should be in the guide and have an example of how to activate it.

Reply:
I _knew_ I was missing something obvious...